### ISSUE TITLE
Steam crashes after starting

### ISSUE THREAD

**Issue by Storm-Engineer** [ValveSoftware/steam-for-linux#4562]
#### Your system information

* Steam client version: 1.0.0.52-2
* Distribution (e.g. Ubuntu): Arch Linux x64
* Opted into Steam client beta?: Yes 
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:

When Steam is started for the first time after booting up the PC it will start up properly, but within a few minutes it will randomly crash. After that, if I try to start Steam again, it will crash after the "logging into account" phase.

First crash after a system boot dump:
https://www.dropbox.com/s/djj7dc2hcovpiuz/crash_20160721175437_1.dmp?dl=0
https://www.dropbox.com/s/ky0ab0i6x05k22p/crash_20160721180953_1.dmp?dl=0
First crash console output:
https://www.dropbox.com/s/k81f9xkz9ipz7ca/storm_stdout-2.txt?dl=0
https://www.dropbox.com/s/t5x2fcl51p3s1et/storm_stdout-3.txt?dl=0
Dumps of subsequent crashes:
https://www.dropbox.com/sh/hqu2f06b8pirant/AADEtPRyR1H5uaqO73k42obPa?dl=0
Console output of subsequent crashes:
https://www.dropbox.com/s/h4lk7ctne94kzgz/storm_stdout.txt?dl=0

The issue randomly started 2 days ago. Earlier the same day I successfully used Steam without any trouble, but it crashes ever since. I did not change anything in my system in between the working and broken states so the only change could have been Steam updating itself.

Since that did a full system upgrade every day as well as force-reinstalling Steam but the issue is still present. Reproducible 100% of the time, making Steam completely broken and unusable for days now!!!

#### Steps for reproducing this issue:

1. Start Steam
2. Wait
3. Steam will crash

There are already a few tickets that are similar to mine (#4551 #4554 #4548 #4545), but they go without any response for several days now while my Steam is is completely broken and I cannot access my $1100 worth of game library so I'm creating this new ticket.

**Comment by werecat** [comment]
Are you opted into any game betas, such as terraria or undertale? If so this might just be a duplicate of #4561 which I have a solution for.

**Comment by NolanSyKinsley** [comment]
This was it! Thank you! Was in the old terraria beta.

**Comment by gdrewb-valve** [comment]
Creating duplicate tickets does not help and in fact takes time away from other things.  As it's not 100% clear that your issue is a duplicate I'll leave this open.

**Comment by Storm-Engineer** [comment]
> Creating duplicate tickets does not help and in fact takes time away from other things. As it's not 100% clear that your issue is a duplicate I'll leave this open.
I could explain that it's not a duplicate until you can prove that my issue is the same as one of those, but I think you answered yourself.

**Comment by Storm-Engineer** [comment]
@werecat Uhh... no idea. I will have to go through all my games and check. I'm not in Terraria beta, that one is sure, and don't have Undertale.

**Comment by Storm-Engineer** [comment]
@werecat @gdrewb-valve OK, I tried if removing all the appmanifest files helps and it did. I then narrowed it down to the file of Ultimate Chicken Horse. Now that I'm thinking about it, I believe I was in Beta for that game, and it doesn1t seem to have a beta anymore. Removing that single appmanifest and then re-downloading the game seems to have fixed the crashes.

So apparently my issue was indeed the same as #4561. Thanks again @werecat ! I guess this now can be closed as a duplicate of that one.
